app/services/idv/vendor_result.rb

```diff
@@ -22,5 +22,15 @@ def initialize(success:, errors:, reasons:, session_id:, normalized_applicant:)
     def success?
       success
     end
+
+    # rubocop:disable Metrics/AbcSize
+    def ==(other)
+      success == other.success &&
+        errors == other.errors &&
+        reasons == other.reasons &&
+        session_id == other.session_id &&
+        normalized_applicant.as_json == other.normalized_applicant.as_json
+    end
+    # rubocop:enable Metrics/AbcSize
```

Comment:
ABC size is "assignment branch condition" complexity and this method scored too high so I just disabled it, because I thought it was more important to have an equality method we could use in specs to compare things